src/Xamarin.Android.Build.Tasks/Tasks/LinkAssembliesNoShrink.cs

```diff
@@ -0,0 +1,98 @@
+using Java.Interop.Tools.Cecil;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using System;
+using System.IO;
+
+namespace Xamarin.Android.Tasks
+{
+	/// <summary>
+	/// This task is for Debug builds where LinkMode=None, LinkAssemblies is for Release builds
+	/// </summary>
+	public class LinkAssembliesNoShrink : Task
```

Comment:
Wouldn't `FixAbstractMethods` be a more descriptive task name here?

Otherwise LGTM.

Reply:
I would agree, however we might add another step later on, so its probably best to keep it as a `Link` named task.